File: Java/Disk.java
public class Disk {

    /* Private Attribute */
    private String type;
    private String capacity;
    private int price;

    /* Constuctor */
    public Disk() {
        
    }

    public Disk(String type, String capacity, int price) {
        this.type = type;
        this.price = price;
    }

    /* Setter dan Getter Name */
    public void setType(String type) {
        this.type = type;
    }

    public String getType() {
        return this.type;
    }

    /* Setter dan Getter Capacity */
    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public String getCapacity() {
        return this.capacity;
    }

    /* Setter dan Getter Price */
    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

}

File: Python/Disk.py
class Disk():

    #private attributes
    __type = "Disk Type"
    __capacity = "Disk Capacity"
    __price = 0

    #Constructor
    def __init__(self, type = "Disk Type", capacity = "Disk Capacity", price = 0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    #Setter dan Getter Type
    def setType(self, type):
        self.__type = type

    def getType(self):
        return self.__type

    #Setter dan Getter Capacity
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity
    
    #Setter dan Getter Price
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price